package.json

```diff
@@ -30,7 +29,7 @@
     "test:eslint": "eslint --config ./.eslintrc.yaml scripts/**/*.js source/js/**/*.js source/js/components/**/*.jsx",
     "test:pug": "pug-lint source/pug/partials/ source/pug/templates/ source/pug/partials/",
     "test:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/components/**/*.scss\" --syntax scss",
-    "test": "npm run build && run-s test:**",
+    "test": "run-s test:**",
```

Comment:
On the fence about this. It won't work properly unless we run `cp env.default .env` on CI before the test task is run.